[2025-05-20 14:45:04,256] INFO: Detected: Unknown (Score: 1.5949)
[2025-05-20 14:45:08,102] INFO: Detected: Unknown (Score: 1.6111)
[2025-05-20 14:46:11,593] INFO: Detected: edwina (Score: 0.6943)
[2025-05-20 14:46:13,197] INFO: Detected: edwina (Score: 0.7139)
[2025-05-20 14:46:14,595] INFO: Detected: edwina (Score: 0.8731)
[2025-05-20 14:46:16,047] INFO: Detected: edwina (Score: 0.7590)
[2025-05-20 14:46:17,544] INFO: Detected: edwina (Score: 0.7548)
[2025-05-20 14:46:19,302] INFO: Detected: edwina (Score: 0.7701)
[2025-05-20 14:46:20,745] INFO: Detected: edwina (Score: 0.7223)
[2025-05-20 14:46:22,202] INFO: Detected: Unknown (Score: 1.0239)
[2025-05-20 14:46:27,630] INFO: Detected: supriya (Score: 0.8191)
[2025-05-20 14:46:55,067] INFO: Detected: Unknown (Score: 1.2815)
[2025-05-20 14:46:56,793] INFO: Detected: Unknown (Score: 1.3751)
[2025-05-20 14:46:58,592] INFO: Detected: Unknown (Score: 1.2410)
[2025-05-20 14:47:00,360] INFO: Detected: Unknown (Score: 1.0573)
[2025-05-20 14:48:37,071] INFO: Detected: Unknown (Score: 1.5585)
[2025-05-20 14:49:50,341] INFO: Detected: Unknown (Score: 1.6167)
[2025-05-20 14:49:51,947] INFO: Detected: Unknown (Score: 1.6282)
[2025-05-20 14:50:37,520] INFO: Detected: Unknown (Score: 1.6505)
[2025-05-20 14:50:39,090] INFO: Detected: Unknown (Score: 1.3520)
[2025-05-20 14:51:18,461] INFO: Detected: Unknown (Score: 1.6162)
[2025-05-20 14:51:44,950] INFO: Detected: Unknown (Score: 1.8711)
[2025-05-20 14:51:45,102] INFO: Detected: Unknown (Score: 1.4878)
[2025-05-20 14:52:04,908] INFO: Detected: Unknown (Score: 1.8352)
[2025-05-20 14:53:34,548] INFO: Detected: Unknown (Score: 1.7320)
[2025-05-20 14:54:10,196] INFO: Detected: Unknown (Score: 1.6583)
[2025-05-20 14:55:23,063] INFO: Detected: Unknown (Score: 1.6651)
[2025-05-20 14:55:48,750] INFO: Detected: Unknown (Score: 1.6212)
[2025-05-20 14:55:54,501] INFO: Detected: Unknown (Score: 1.6529)
[2025-05-20 14:57:23,040] INFO: Detected: edwina (Score: 0.7826)
[2025-05-20 14:57:24,833] INFO: Detected: edwina (Score: 0.6819)
[2025-05-20 14:57:26,503] INFO: Detected: edwina (Score: 0.7671)
[2025-05-20 14:57:28,203] INFO: Detected: edwina (Score: 0.7126)
[2025-05-20 14:57:29,894] INFO: Detected: edwina (Score: 0.6721)
[2025-05-20 14:57:31,663] INFO: Detected: edwina (Score: 0.7434)
[2025-05-20 14:57:33,705] INFO: Detected: edwina (Score: 0.6669)
[2025-05-20 14:57:35,757] INFO: Detected: edwina (Score: 0.6716)
[2025-05-20 14:57:38,011] INFO: Detected: edwina (Score: 0.6872)
[2025-05-20 14:57:39,977] INFO: Detected: edwina (Score: 0.8860)
[2025-05-20 14:57:43,660] INFO: Detected: edwina (Score: 0.9716)
[2025-05-20 14:57:45,509] INFO: Detected: Unknown (Score: 1.0049)
[2025-05-20 14:58:22,619] INFO: Detected: Unknown (Score: 1.4120)
[2025-05-20 14:58:26,098] INFO: Detected: Unknown (Score: 1.5329)
[2025-05-20 14:58:28,897] INFO: Detected: Unknown (Score: 1.4098)
[2025-05-20 14:58:36,328] INFO: Detected: Unknown (Score: 1.3770)
[2025-05-20 15:00:00,493] INFO: Detected: Unknown (Score: 1.5621)
[2025-05-20 15:00:02,783] INFO: Detected: Unknown (Score: 1.7529)
[2025-05-20 15:00:03,170] INFO: Detected: Unknown (Score: 1.7873)
[2025-05-20 15:00:10,786] INFO: Detected: Unknown (Score: 1.4480)
[2025-05-20 15:00:12,699] INFO: Detected: Unknown (Score: 1.3587)
[2025-05-20 15:00:23,924] INFO: Detected: Unknown (Score: 1.0388)
[2025-05-20 15:00:25,466] INFO: Detected: shravani (Score: 0.8772)
[2025-05-20 15:00:31,244] INFO: Detected: shravani (Score: 0.5702)
[2025-05-20 15:01:51,823] INFO: Detected: Unknown (Score: 1.7247)
[2025-05-20 15:01:54,164] INFO: Detected: Unknown (Score: 1.9494)
[2025-05-20 15:01:55,773] INFO: Detected: Unknown (Score: 1.0278)
[2025-05-20 15:06:08,081] INFO: Detected: Unknown (Score: 1.8310)
[2025-05-20 15:06:34,936] INFO: Detected: Unknown (Score: 1.6874)
[2025-05-20 15:07:23,022] INFO: Detected: Unknown (Score: 1.9051)
[2025-05-20 15:07:34,151] INFO: Detected: Unknown (Score: 1.9227)
[2025-05-20 15:07:54,633] INFO: Detected: Unknown (Score: 1.7032)
[2025-05-20 15:08:06,786] INFO: Detected: Unknown (Score: 1.8211)
[2025-05-20 15:08:15,546] INFO: Detected: Unknown (Score: 1.7466)
[2025-05-20 15:11:26,881] INFO: Detected: Unknown (Score: 1.6008)
[2025-05-20 15:11:27,049] INFO: Detected: Unknown (Score: 1.3532)
[2025-05-20 15:11:28,763] INFO: Detected: Unknown (Score: 1.3997)
[2025-05-20 15:11:28,963] INFO: Detected: Unknown (Score: 1.7766)
[2025-05-20 15:11:30,666] INFO: Detected: Unknown (Score: 1.3693)
[2025-05-20 15:11:30,779] INFO: Detected: Unknown (Score: 1.4366)
[2025-05-20 15:11:32,357] INFO: Detected: Unknown (Score: 1.2754)
[2025-05-20 15:11:33,982] INFO: Detected: Unknown (Score: 1.5360)
[2025-05-23 09:38:32,546] WARNING: Not Found: /favicon.ico
[2025-05-23 09:38:54,863] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:38:57,248] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:38:58,519] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:38:59,893] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:01,268] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:02,790] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:04,009] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:06,316] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:07,516] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:08,574] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:09,775] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:11,041] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:12,374] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:13,831] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:15,140] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:16,607] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:17,996] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:20,247] INFO: Detected: Unknown (Score: 1.6669)
[2025-05-23 09:39:50,473] INFO: Detected: Unknown (Score: 1.8738)
[2025-05-23 09:39:54,338] INFO: Detected: Unknown (Score: 1.8658)
[2025-05-23 09:39:56,574] INFO: Detected: Unknown (Score: 1.8967)
[2025-05-23 09:41:25,097] INFO: Detected: Unknown (Score: 1.8245)
[2025-05-23 09:41:39,009] INFO: Detected: Unknown (Score: 1.9857)
[2025-05-28 11:29:28,160] WARNING: Not Found: /favicon.ico
[2025-05-28 11:29:48,193] ERROR: Internal Server Error: /attendance/logs/
Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py", line 181, in attendance_logs
    return render(request, 'attendance_logs.html', {'logs': logs})
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: attendance_logs.html
[2025-05-28 11:31:27,436] ERROR: Internal Server Error: /attendance/logs/
Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py", line 181, in attendance_logs
    return render(request, 'attendance_logs.html', {'logs': logs})
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: attendance_logs.html
[2025-05-28 11:39:44,690] INFO: C:\Users\Admin\Downloads\face_recognition_admin\face_recognition_admin\settings.py changed, reloading.
[2025-05-28 11:43:11,571] INFO: [ATTENDANCE] shravani entered at 2025-05-28 11:43:11.571581
[2025-05-28 11:43:14,942] INFO: [ATTENDANCE] supriya entered at 2025-05-28 11:43:14.942916
[2025-05-28 14:50:59,214] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\attendance_utils.py changed, reloading.
[2025-05-28 14:55:31,580] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\attendance_utils.py changed, reloading.
[2025-05-28 14:56:19,855] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py changed, reloading.
[2025-05-28 14:58:55,449] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\urls.py changed, reloading.
[2025-05-28 15:27:19,729] INFO: [ATTENDANCE] supriya entered at 2025-05-28 15:27:19.729845
[2025-05-28 15:54:41,665] INFO: [ATTENDANCE] supriya entered at 2025-05-28 15:54:41.665786
[2025-05-28 16:01:43,836] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\attendance_utils.py changed, reloading.
[2025-05-28 16:05:24,532] INFO: [ATTENDANCE] supriya entered at 2025-05-28 16:05:24.532756
[2025-05-28 16:07:42,093] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\attendance_utils.py changed, reloading.
[2025-05-28 16:09:36,530] INFO: [ATTENDANCE] supriya entered at 2025-05-28 16:09:36.530127
[2025-05-28 16:09:36,686] ERROR: Error logging attendance for supriya: (1048, "Column 'in_time' cannot be null")
[2025-05-28 16:09:38,174] ERROR: Error logging attendance for supriya: (1048, "Column 'in_time' cannot be null")
[2025-05-28 16:09:39,721] ERROR: Error logging attendance for supriya: (1048, "Column 'in_time' cannot be null")
[2025-05-28 16:18:59,503] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\attendance_utils.py changed, reloading.
[2025-05-28 16:27:53,560] INFO: [ATTENDANCE] supriya entered at 2025-05-28 16:27:53.560314
[2025-05-28 16:27:53,888] ERROR: Error logging attendance for supriya: (1048, "Column 'in_time' cannot be null")
[2025-05-28 16:28:19,188] ERROR: Error logging attendance for supriya: (1048, "Column 'in_time' cannot be null")
[2025-05-28 16:37:40,966] INFO: [ATTENDANCE] supriya entered at 2025-05-28 16:37:40.966283
[2025-05-28 16:37:41,294] INFO: [ATTENDANCE] supriya entered at 2025-05-28 11:07:40.997531+00:00
[2025-05-28 16:47:13,123] INFO: [ATTENDANCE] supriya entered at 2025-05-28 16:47:13.123570
[2025-05-28 16:47:13,186] INFO: [ATTENDANCE] supriya updated at 2025-05-28 11:17:13.186051+00:00
[2025-05-28 16:47:14,974] INFO: [ATTENDANCE] supriya updated at 2025-05-28 11:17:14.974799+00:00
[2025-05-29 09:52:00,849] WARNING: Not Found: /favicon.ico
[2025-05-29 09:54:21,119] INFO: [ATTENDANCE] supriya entered at 2025-05-29 09:54:21.119190
[2025-05-29 09:54:23,244] INFO: [ATTENDANCE] supriya entered at 2025-05-29 04:24:21.650416+00:00
[2025-05-29 09:54:26,960] INFO: [ATTENDANCE] supriya updated at 2025-05-29 04:24:26.960480+00:00
[2025-05-29 10:48:42,996] INFO: [ATTENDANCE] supriya entered at 2025-05-29 10:48:42.996208
[2025-05-29 10:48:43,136] INFO: [ATTENDANCE] supriya updated at 2025-05-29 05:18:43.089950+00:00
[2025-05-29 10:48:45,429] INFO: [ATTENDANCE] supriya updated at 2025-05-29 05:18:45.429242+00:00
[2025-05-29 16:51:58,077] WARNING: Not Found: /favicon.ico
[2025-05-30 09:32:51,519] WARNING: Not Found: /favicon.ico
[2025-05-30 09:34:32,005] INFO: [ATTENDANCE] supriya entered at 2025-05-30 09:34:32.005748
[2025-05-30 09:34:32,568] INFO: [ATTENDANCE] supriya entered at 2025-05-30 04:04:32.068244+00:00
[2025-05-30 10:38:11,299] ERROR: Internal Server Error: /attendance/summary/
Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py", line 210, in attendance_summary
    if record.in_time and record.in_time.time() > time(10, 0):  # Late if after 10:00 AM
TypeError: 'module' object is not callable
[2025-05-30 10:45:43,292] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py changed, reloading.
[2025-05-30 10:51:58,361] ERROR: Internal Server Error: /attendance/summary/
Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py", line 208, in attendance_summary
    if record.in_time and record.in_time.time() > time(10, 0):
TypeError: 'module' object is not callable
[2025-05-30 10:58:34,254] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py changed, reloading.
[2025-05-30 11:38:07,616] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py changed, reloading.
[2025-05-30 11:59:46,678] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\urls.py changed, reloading.
[2025-05-30 12:01:52,707] ERROR: Internal Server Error: /attendance/summary/
Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py", line 175, in attendance_summary
    if record.in_time and record.in_time.time() > time(10, 0):
TypeError: 'module' object is not callable
[2025-05-30 12:11:32,334] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py changed, reloading.
[2025-05-30 12:14:06,256] ERROR: Internal Server Error: /attendance/summary/
Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py", line 181, in attendance_summary
    if record.in_time and record.in_time.time() > time(10, 0):
TypeError: 'module' object is not callable
[2025-05-30 14:47:13,439] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py changed, reloading.
[2025-05-30 14:53:02,531] ERROR: Internal Server Error: /attendance/export/pdf/
Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py", line 207, in export_attendance_pdf
    queryset, from_date, to_date = get_attendance_summary_queryset(request)
  File "C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py", line 165, in get_attendance_summary_queryset
    queryset = queryset.filter(date__range=[from_date, to_date])
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\query.py", line 1481, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\query.py", line 1499, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\query.py", line 1506, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\sql\query.py", line 1643, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\sql\query.py", line 1675, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\sql\query.py", line 1585, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\sql\query.py", line 1412, in build_lookup
    lookup = lookup_class(lhs, rhs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\lookups.py", line 38, in __init__
    self.rhs = self.get_prep_lookup()
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\lookups.py", line 321, in get_prep_lookup
    rhs_value = self.lhs.output_field.get_prep_value(rhs_value)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\fields\__init__.py", line 1537, in get_prep_value
    return self.to_python(value)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\fields\__init__.py", line 1503, in to_python
    raise exceptions.ValidationError(
django.core.exceptions.ValidationError: ['“None” value has an invalid date format. It must be in YYYY-MM-DD format.']
[2025-05-30 14:53:17,911] ERROR: Internal Server Error: /attendance/export/excel/
Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py", line 219, in export_attendance_excel
    queryset, from_date, to_date = get_attendance_summary_queryset(request)
  File "C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py", line 165, in get_attendance_summary_queryset
    queryset = queryset.filter(date__range=[from_date, to_date])
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\query.py", line 1481, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\query.py", line 1499, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\query.py", line 1506, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\sql\query.py", line 1643, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\sql\query.py", line 1675, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\sql\query.py", line 1585, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\sql\query.py", line 1412, in build_lookup
    lookup = lookup_class(lhs, rhs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\lookups.py", line 38, in __init__
    self.rhs = self.get_prep_lookup()
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\lookups.py", line 321, in get_prep_lookup
    rhs_value = self.lhs.output_field.get_prep_value(rhs_value)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\fields\__init__.py", line 1537, in get_prep_value
    return self.to_python(value)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\django\db\models\fields\__init__.py", line 1503, in to_python
    raise exceptions.ValidationError(
django.core.exceptions.ValidationError: ['“None” value has an invalid date format. It must be in YYYY-MM-DD format.']
[2025-05-30 15:04:28,264] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py changed, reloading.
[2025-05-30 15:12:41,865] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py changed, reloading.
[2025-05-30 17:24:55,317] WARNING: Not Found: /.well-known/appspecific/com.chrome.devtools.json
[2025-05-30 17:37:00,534] INFO: [ATTENDANCE] supriya entered at 2025-05-30 17:37:00.534374
[2025-05-30 17:37:00,549] INFO: [ATTENDANCE] supriya updated at 2025-05-30 12:07:00.549997+00:00
[2025-05-30 17:37:03,647] INFO: [ATTENDANCE] supriya updated at 2025-05-30 12:07:03.647691+00:00
[2025-05-30 17:37:06,376] INFO: [ATTENDANCE] supriya updated at 2025-05-30 12:07:06.360600+00:00
[2025-05-30 17:37:08,137] INFO: [ATTENDANCE] supriya updated at 2025-05-30 12:07:08.137073+00:00
[2025-06-02 09:22:46,397] WARNING: Not Found: /favicon.ico
[2025-06-02 09:24:38,659] INFO: [ATTENDANCE] supriya entered at 2025-06-02 09:24:38.659961
[2025-06-02 09:24:39,269] INFO: [ATTENDANCE] supriya entered at 2025-06-02 03:54:38.675582+00:00
[2025-06-02 14:41:51,423] INFO: [ATTENDANCE] edwina entered at 2025-06-02 14:41:51.342461
[2025-06-02 14:41:52,779] INFO: [ATTENDANCE] edwina entered at 2025-06-02 09:11:52.135037+00:00
[2025-06-03 09:14:13,783] WARNING: Not Found: /favicon.ico
[2025-06-03 09:15:33,160] INFO: [ATTENDANCE] supriya entered at 2025-06-03 09:15:33.160115
[2025-06-03 09:15:34,191] INFO: [ATTENDANCE] supriya entered at 2025-06-03 03:45:33.175741+00:00
[2025-06-03 09:54:01,966] INFO: [ATTENDANCE] edwina entered at 2025-06-03 09:54:01.950845
[2025-06-03 09:54:02,732] INFO: [ATTENDANCE] edwina entered at 2025-06-03 04:24:02.294573+00:00
[2025-06-03 09:54:07,474] INFO: [ATTENDANCE] edwina updated at 2025-06-03 04:24:07.474678+00:00
[2025-06-03 16:18:43,607] WARNING: Not Found: /favicon.ico
[2025-06-03 17:30:52,550] INFO: [ATTENDANCE] edwina entered at 2025-06-03 17:30:52.550681
[2025-06-03 17:30:52,800] INFO: [ATTENDANCE] edwina updated at 2025-06-03 12:00:52.722541+00:00
[2025-06-03 17:30:54,571] INFO: [ATTENDANCE] edwina updated at 2025-06-03 12:00:54.571686+00:00
[2025-06-03 17:31:44,919] INFO: [ATTENDANCE] supriya entered at 2025-06-03 17:31:44.919522
[2025-06-03 17:31:45,302] INFO: [ATTENDANCE] supriya updated at 2025-06-03 12:01:45.200816+00:00
[2025-06-04 09:51:33,033] WARNING: Not Found: /favicon.ico
[2025-06-04 09:52:06,587] INFO: [ATTENDANCE] supriya entered at 2025-06-04 09:52:06.587292
[2025-06-04 09:52:07,915] INFO: [ATTENDANCE] supriya entered at 2025-06-04 04:22:06.602916+00:00
[2025-06-04 17:29:50,319] WARNING: Not Found: /favicon.ico
[2025-06-04 17:30:38,879] INFO: [ATTENDANCE] supriya entered at 2025-06-04 17:30:38.879500
[2025-06-04 17:30:39,441] INFO: [ATTENDANCE] supriya updated at 2025-06-04 12:00:39.176388+00:00
[2025-06-05 09:31:35,299] WARNING: Not Found: /favicon.ico
[2025-06-05 09:32:45,597] INFO: [ATTENDANCE] supriya entered at 2025-06-05 09:32:45.597437
[2025-06-05 09:32:46,659] INFO: [ATTENDANCE] supriya entered at 2025-06-05 04:02:45.722427+00:00
[2025-06-05 09:58:58,688] WARNING: Not Found: /favicon.ico
[2025-06-05 16:35:19,053] WARNING: Not Found: /favicon.ico
[2025-06-05 16:36:18,219] INFO: [ATTENDANCE] supriya entered at 2025-06-05 16:36:18.219440
[2025-06-05 16:36:18,297] INFO: [ATTENDANCE] supriya updated at 2025-06-05 11:06:18.235061+00:00
[2025-06-05 17:00:51,024] INFO: [ATTENDANCE] supriya entered at 2025-06-05 17:00:51.024568
[2025-06-05 17:00:51,055] INFO: [ATTENDANCE] supriya updated at 2025-06-05 11:30:51.055813+00:00
[2025-06-05 17:00:52,661] INFO: [ATTENDANCE] supriya updated at 2025-06-05 11:30:52.661698+00:00
[2025-06-05 17:56:35,373] WARNING: Not Found: /favicon.ico
[2025-06-05 18:00:43,888] INFO: [ATTENDANCE] supriya entered at 2025-06-05 18:00:43.888103
[2025-06-05 18:00:43,919] INFO: [ATTENDANCE] supriya updated at 2025-06-05 12:30:43.888103+00:00
[2025-06-06 09:26:38,933] WARNING: Not Found: /favicon.ico
[2025-06-06 09:27:54,735] INFO: [ATTENDANCE] supriya entered at 2025-06-06 09:27:54.735874
[2025-06-06 09:27:55,220] INFO: [ATTENDANCE] supriya entered at 2025-06-06 03:57:54.751498+00:00
[2025-06-06 09:27:56,820] INFO: [ATTENDANCE] supriya updated at 2025-06-06 03:57:56.820116+00:00
[2025-06-06 10:17:39,901] INFO: [ATTENDANCE] supriya entered at 2025-06-06 10:17:39.901607
[2025-06-06 10:17:39,917] INFO: [ATTENDANCE] supriya updated at 2025-06-06 04:47:39.917231+00:00
[2025-06-06 15:57:33,971] INFO: C:\Users\Admin\Downloads\face_recognition_admin\faceapp\views.py changed, reloading.
[2025-06-06 18:00:03,549] INFO: [ATTENDANCE] supriya entered at 2025-06-06 18:00:03.549515
[2025-06-06 18:00:03,612] INFO: [ATTENDANCE] supriya updated at 2025-06-06 12:30:03.612010+00:00
[2025-06-06 18:00:05,222] INFO: [ATTENDANCE] supriya updated at 2025-06-06 12:30:05.222912+00:00
[2025-06-09 09:33:34,589] WARNING: Not Found: /favicon.ico
[2025-06-09 09:36:23,632] INFO: [ATTENDANCE] supriya entered at 2025-06-09 09:36:23.632642
[2025-06-09 09:36:24,179] INFO: [ATTENDANCE] supriya entered at 2025-06-09 04:06:23.632642+00:00
[2025-06-09 09:36:26,880] INFO: [ATTENDANCE] supriya updated at 2025-06-09 04:06:26.880511+00:00
[2025-06-09 17:36:42,823] INFO: [ATTENDANCE] supriya entered at 2025-06-09 17:36:42.823935
[2025-06-09 17:36:43,542] INFO: [ATTENDANCE] supriya updated at 2025-06-09 12:06:43.292690+00:00
[2025-06-10 09:05:50,863] WARNING: Not Found: /favicon.ico
[2025-06-10 09:06:17,685] INFO: [ATTENDANCE] supriya entered at 2025-06-10 09:06:17.685360
[2025-06-10 09:06:17,966] INFO: [ATTENDANCE] supriya entered at 2025-06-10 03:36:17.685360+00:00
[2025-06-10 17:36:49,901] INFO: [ATTENDANCE] supriya entered at 2025-06-10 17:36:49.901991
[2025-06-10 17:36:49,995] INFO: [ATTENDANCE] supriya updated at 2025-06-10 12:06:49.948860+00:00
[2025-06-10 17:36:51,401] INFO: [ATTENDANCE] supriya updated at 2025-06-10 12:06:51.401952+00:00
[2025-06-10 17:42:55,667] WARNING: Not Found: /favicon.ico
